osbuild/util/rhsm.py

```diff
@@ -38,6 +38,20 @@ def get_fallback_rhsm_secrets(self):
 
         raise RuntimeError("no matching rhsm key and cert")
 
+    @staticmethod
+    def get_consumer_secrets():
+        """Returns the consumer identity certificate which uniquely identifies the system"""
+        key = "/etc/pki/consumer/cert.pem"
+        cert = "/etc/pki/consumer/key.pem"
+
+        if not (os.path.exists(key) and os.path.exists(cert)):
+            raise RuntimeError("rhsm consumer key and cert not found")
```

Comment:
Not in this PR, but in the future we should probably have a custom exception so that we can report these kind of errors in a machine readable way back to Composer.

Reply:
completely agree